tests/idris2/perf008/run

```diff
@@ -0,0 +1,3 @@
+$1 --check FinPerf.idr
+
```

Comment:
Don't you want to run the program too? Otherwise what's the point of the `main`?

Reply:
I thought the issue was during typechecking, will change the code regardless.

Reply:
Changed to `--exec main` instead of `--check`. Same performance (roughly 2 seconds for N=500)